controls/radgridview/export/how-to/modify-exported-content.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Modify Content Before Exporting
+page_title: ExportFormat.Xlsx
+description: ExportFormat.Xlsx
+slug: gridview-export-xlsx
+tags: exportformat.xlsx
+published: True
+position: 7
+---
+
+# Modify RadGridView`s Content Before Exporting
+
+In __R1 2016__, we have introduced two new extension methods related to the exporting of RadGridView - *ExportToWorkBook()* and *ExportToRadFixedDocument()*. You can use them if you need to modify the content of the exported RadGridView and avoid styling the document manually.
```

Comment:
Add info that those utilize SpreadProcessing and PdfProcessing and links to the libraries' documentation. That way customers will be able to find how to modify whatever they like.